Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    timespan = TimeSpan(start=datetime.now() - timedelta(days=2))
</pre><p>Think it would be good to derive the time range from the incident. Something like this</p><pre class="ql-syntax" spellcheck="false">timespan = TimeSpan(
&nbsp; &nbsp; start=pd.to_datetime(incident_details.iloc[0]["properties.firstActivityTimeUtc"]) - timedelta(days=1),
&nbsp; &nbsp; end=pd.to_datetime(incident_details.iloc[0]["properties.lastActivityTimeUtc"]) + timedelta(days=1)
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>